server/analysis/gpr/optimize.py

```diff
@@ -25,6 +32,20 @@ def __init__(self, ypreds=None, sigmas=None, minl=None, minl_conf=None):
         self.minl_conf = minl_conf
 
 
+def gpflow_predict(model, Xin):
```

Comment:
It is a little confusing to put the predict function in optimize.py. We may want a new file called predict.py